routes/shops.js

```diff
@@ -3,6 +3,6 @@ const router = express.Router();
 ///////
 const controller = require("../controllers/shops");
 
-router.post("/", controller.create);
+router.route("/").post(controller.create).get(controller.get);
```

Comment:
@Behramkhann , you must always include `shopId` after `/shops` whenever you want to get a single shop.
```suggestion
router.post("/", controller.create);
router.route(":/shopId").get(controller.get);
```